### PR TITLE
Registation page tweaks

### PR DESCRIPTION
**Pull request by fat23cat**

### CODE REVIEW

File: ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.html [hunk 1]
```diff
@@ -1,31 +1,24 @@
...
-                    <input matInput
-                           formControlName="zip"
-                           minlength="5"
-                           maxlength="5"
-                           placeholder="5 digit zip"
-                           pattern="[0-9]{5}">
+                <p class="invitation-form__question bold">What is your current mailing zip code?</p>
+                <mat-form-field class="invitation-form__input" Label="always">
+                    <input matInput formControlName="zip" minlength="5" maxlength="5"
+                        placeholder="Enter mailing zip code" pattern="[0-9]{5}">
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.html:19]: very minor thing. I have a preference to laying out the attributes in elements like this vertically. I find it a lot easier to read and notice what is going on. Also easier to do merges.

File: ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.scss [hunk 2]
```diff
@@ -1,38 +1,39 @@
...
 }
 
 .invitation-form {
     display: flex;
     flex-direction: column;
 
+    &__title {
+        color: #454545;
+    }
+
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.scss:12]: The class name naming is interesting. How did you come up with this? I think it makes a lot of sense.

**Review by MocanaAtBroad — APPROVED** [review]

File: ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.html (hunk 1 above)
Comment (fat23cat) [reply ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.html:19]: Autoformatting code betrayed me :)

File: ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.scss (hunk 2 above)
Comment (fat23cat) [reply ddp-workspace/projects/ddp-testboston/src/app/components/user-registration-prequal/user-registration-prequal.component.scss:12]: I used BEM methodology and SCSS syntax.

**Pull request merged**